decentraland/unity-explorer
# chore: AB debug tools

## What does this PR change?

- Its no longer necessary to specify the hash of the remote entity to get the asset bundles. Its fetched automatically
- Now AB have further info in the hierarchy. They have the version of the manifest and the source they were loaded from.

## How to test the changes?

1. Launch the explorer
2. Check that it starts. This is a debug tool for localhost, the regular flow should not have been invaded

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
